client/src/views/Home/Home.jsx

```diff
@@ -2,34 +2,43 @@ import React, { useState } from "react";
 
 import CreateGame from "./CreateParty";
 import JoinParty from "./JoinParty";
+import HowToPlay from "./HowToPlay";
 import { Modal } from "../../components";
 
 function Home({ username: storageUsername, slug, onJoinParty }) {
   const [createGameOpen, setCreateGameOpen] = useState(false);
+  const [howToPlayOpen, setHowToPlayOpen] = useState(false);
   const [username, setUsername] = useState(storageUsername.slice(0, -7));
   const [errorMessage, setErrorMessage] = useState("");
 
   const showCreateGameView = () => setCreateGameOpen(true);
   const hideCreateGameView = () => setCreateGameOpen(false);
 
+  // Show and hide content about how to play with charades app
+  const showHowToPlay = () => setHowToPlayOpen(true);
+  const hideHowToPlay = () => setHowToPlayOpen(false);
```

Comment:
when passed down as props too we change the name. So when passed as props it would be:

`
onHowToPlayModalOpen={handleHowToPlayModalOpen}
`

This way we know where a function originates (and just to have some consistency in naming)